rules/python_rules.build_defs

```diff
@@ -493,23 +494,25 @@ def python_wheel(name:str, version:str, hashes:list=None, package_name:str=None,
     if name_scheme:
         urls += [name_scheme.format(url_base=url_base,
                                     package_name=package_name,
+                                    initial=initial,
                                     version=version)]
     elif CONFIG.PYTHON_WHEEL_NAME_SCHEME:
-        urls += [CONFIG.PYTHON_WHEEL_NAME_SCHEME.format(url_base=url_base,
-                                                        package_name=package_name,
-                                                        version=version)]
+        urls += [scheme.format(url_base=url_base, package_name=package_name, initial=initial, version=version)
+                 for scheme in CONFIG.PYTHON_WHEEL_NAME_SCHEME]
     else:
         # Populate urls using a reasonable set of possible wheel naming schemes.
         # Look for an arch-specific wheel first; in some cases there can be both (e.g. protobuf
         # has optional arch-specific bits) and we prefer the one with the cool stuff.
         urls += ['{url_base}/{package_name}-{version}-${{OS}}-${{ARCH}}.whl'.format(url_base=url_base,
                                                                                     package_name=package_name,
+                                                                                    initial=initial,
```

Comment:
It's no immediately obvious what initial is or why it's useful. Looks like it ends up as the first character of package_name?

Reply:
That is exactly what it is. There are wheels stored on PyPI in a predictable scheme but it involves the first initial, so we have to make that available to the scheme template.